ccd-definition/CaseEventToComplexTypes/DefendantResponseLRspec.json

```diff
@@ -130,50 +131,74 @@
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
+    "CaseFieldID": "respondent1DQHearingFastClaim",
     "ListElementCode": "hearingLength",
     "FieldDisplayOrder": 1,
     "DisplayContext": "MANDATORY"
   },
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
+    "CaseFieldID": "respondent1DQHearingFastClaim",
     "ListElementCode": "hearingLengthHours",
     "FieldDisplayOrder": 2,
     "DisplayContext": "MANDATORY"
   },
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
+    "CaseFieldID": "respondent1DQHearingFastClaim",
     "ListElementCode": "hearingLengthDays",
     "FieldDisplayOrder": 3,
     "DisplayContext": "MANDATORY"
   },
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
+    "CaseFieldID": "respondent1DQHearingFastClaim",
     "ListElementCode": "unavailableDatesRequired",
     "FieldDisplayOrder": 4,
     "DisplayContext": "MANDATORY"
   },
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
-    "ListElementCode": "unavailableDates.who",
+    "CaseFieldID": "respondent1DQHearingFastClaim",
+    "ListElementCode": "unavailableDatesLRspec.who",
     "FieldDisplayOrder": 5,
     "DisplayContext": "MANDATORY"
   },
   {
     "ID": "Hearing",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "respondent1DQHearing",
-    "ListElementCode": "unavailableDates.date",
-    "FieldDisplayOrder": 6,
-    "DisplayContext": "MANDATORY"
+    "CaseFieldID": "respondent1DQHearingFastClaim",
+    "ListElementCode": "unavailableDatesLRspec.date",
+    "FieldDisplayOrder": 8,
```

Comment:
why did the "FieldDisplayOrder" change from 6 to 8? also there is another "FieldDisplayOrder" with rank 8 down below.

Reply:
because I want to display that element on 8 rather than 6.

Reply:
have a look at number again

Reply:
change incorporated